px-vis-behavior-d3.html

```diff
@@ -1624,7 +1624,7 @@
           //for each datapoint, iterate through the dimensions
           return path(dims.map(function(p) {
             //check that we have data; if not, return false so it doesnt draw a line
-            if(!d || (!d[p] && d[p] !== 0)) {
+            if(!d || (!d[p] && d[p] !== 0) || !this.y[p]) {
```

Comment:
Yeah, it was throwing console errors when we are deleting a tag from the chart, because this.completeSeriesConfig has deleted tag config but this.y does not have it.

Reply:
it seems like timing issue only, will look into it more to find out a better solution. Not sure if we should merge it now or not.

Reply:
@benoitjchevalier should it be ok to merge it now for the upcoming APM relaes and see if degrades the performance if any. Meantime, we will see if we have an alternate solution and change it back when we do.